src/agent/src/image.rs

```diff
@@ -5,9 +5,20 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+use std::collections::HashMap;
+use std::env;
+use std::fs;
+use std::path::Path;
+use std::sync::Arc;
+
 use anyhow::{anyhow, Result};
+use image_rs::image::ImageClient;
 use tokio::sync::Mutex;
 
+use crate::rpc::CONTAINER_BASE;
+
+const KATA_CC_IMAGE_WORK_DIR: &str = "/run/image/";
```

Comment:
Let's remove the `CC` part on names like this, if you agree.

Reply:
Sure. Thanks.

Reply:
Fixed.